CHANGES.rst

```diff
@@ -14,6 +14,7 @@ ramp_fitting
 ------------
 
 - Added ramp fitting code [#6]
+- Added DQ flag parameter to `ramp_fit` [#25]
```

Comment:
Start a new section in the `Changes.rst` and move the log there.
```
0.2.2 (Unreleased)
==================

ramp_fitting
------------
```

Reply:
Made these changes to the change log.